core/src/apps/bitcoin/keychain.py

```diff
@@ -37,6 +39,21 @@ def get_namespaces_for_coin(coin: coininfo.CoinInfo):
         # BIP-84 - native segwit: m/84'/slip44' (/account'/change/addr)
         namespaces.append([84 | HARDENED, slip44_id])
 
+    if coin.coin_name in BITCOIN_NAMES:
+        # compatibility namespace for Casa
```

Comment:
Shall we add info to the comment such as:

"introduced in July 2020, remove by July 2021"?